news/create-newsfile.py

```diff
@@ -52,22 +75,24 @@ def create_block(block_name, files):
     return block
 
 
-def get_news_version():
-    proc = Popen(r'git show HEAD:NEWS.md'.split(), cwd=str(root_dir), stdout=PIPE)
-    stdout, _ = proc.communicate()
-    stdout = stdout.decode()
+def get_last_version():
+    stdout = _exec(r'git show HEAD:NEWS.md')
     return stdout[:stdout.index('\n')]
 
 
 def create_version():
-    version = (root_dir / 'VERSION').read_text().rstrip()
-    news_version = get_news_version()
-    if version == news_version:
-        print('VERSION file contains the same version as the current NEWS.md file.\n' \
-              'Probably you are trying to create the newsfile before bumping the `VERSION` file.\n' \
+    global next_version, last_version
+
+    next_version = (root_dir / 'VERSION').read_text().rstrip()
```

Comment:
I am just wondering why we need `global`. As I see the calculation of both `next_version` and `last_version` is pure:
nor the content of the version file, nor the git tree changes. Instead of a global variable, they could be a "getter" functions.

Reply:
Actually `next_version` might be set by the user interactively, if the `VERSION` file is not bumped.
I removed the interactive feature, as we won't run this script manually anymore.

I added getters, as you advised.